interface/wx/translation.h

```diff
@@ -124,21 +143,38 @@ class wxTranslations
 
         @return @true if a suitable catalog was found, @false otherwise
 
-        @see AddCatalog()
+        @see AddAvailableCatalog()
      */
     bool AddStdCatalog();
 
     /**
         Add a catalog for use with the current locale.
 
-        By default, it is searched for in standard places (see
+        By default, the catalog is searched for in standard places (see
         wxFileTranslationsLoader), but you may also prepend additional
         directories to the search path with
         wxFileTranslationsLoader::AddCatalogLookupPathPrefix().
 
         All loaded catalogs will be used for message lookup by GetString() for
         the current locale.
 
+        @return
```

Comment:
I'd explicitly mention the msgIdLanguage/en situation here too - i.e. note that it is normal with some languages (typically English) that there's no catalog.